src/scss/tools/mixins/_border.scss

```diff
@@ -0,0 +1,14 @@
+//
+// ==========================================================================
+// Border mixin
+// Used in bordered utility class to add padding, border and border
+// radius to element
+// ==========================================================================
+//
+// @include bordered();
+
+@mixin bordered() {
```

Comment:
It's worth using some defaults to set the border size, colour, spacing and radius. So we can reuse this in other scenarios.